src/systems/physics/Physics.cc

```diff
@@ -173,8 +173,10 @@ class gz::sim::systems::PhysicsPrivate
   public: struct MinimumFeatureList : physics::FeatureList<
           physics::FindFreeGroupFeature,
           physics::SetFreeGroupWorldPose,
+          physics::Gravity,
```

Comment:
This will break ABI. We can only release it in Kura. Please retarget this PR to main.